device/futhark/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ traccc_add_library(
     traccc_futhark
     futhark
     TYPE SHARED
+    "src/cca.cpp"
```

Comment:
I would prefer if the header files were listed as well, but it's not mandatory...

Reply:
Headers added.